google-http-client/src/test/java/com/google/api/client/http/javanet/NetHttpTransportTest.java

```diff
@@ -36,6 +37,25 @@
     "GET", "POST", "HEAD", "OPTIONS", "PUT", "DELETE", "TRACE"
   };
 
+  public void testMtls() throws Exception {
+    KeyStore trustStore = KeyStore.getInstance("JKS");
+    KeyStore keyStore = KeyStore.getInstance("PKCS12");
+    keyStore.load(
+        this.getClass()
+            .getClassLoader()
+            .getResourceAsStream("com/google/api/client/util/secret.p12"),
+        "notasecret".toCharArray());
+
+    NetHttpTransport transport =
+        new NetHttpTransport.Builder().trustCertificates(trustStore).build();
+    assertFalse(transport.isMtls());
+    transport =
+        new NetHttpTransport.Builder()
+            .trustCertificates(trustStore, keyStore, "notasecret")
+            .build();
+    assertTrue(transport.isMtls());
```

Comment:
These 2 tests should be split into separate tests and the test name should indicate what condition it's trying to test.

Reply:
done